playbooks/roles/unbound/templates/cache-and-forward.conf.j2

```diff
@@ -0,0 +1,26 @@
+#jinja2: lstrip_blocks: "True"
```

Comment:
Why is this commented out?

Reply:
@UmanShahzad, this is the way to specify the jinja2 configuration directive.